web/themes/custom/hatter/hatter.theme

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * @file
+ * Functions to support theming in the hatter theme, copied from Bartik as a base.
```

Comment:
Stable doesn't provide it's own .theme file so I picked Bartik as a starting point since I'm new to d8 theming, so do want this one in. Also, it was a bit last minute as you wanted me to commit my branch so that you could merge and add in the butler output bits.